src/joystick/windows/SDL_windows_gaming_input.c

```diff
@@ -233,7 +233,7 @@ static HRESULT STDMETHODCALLTYPE IEventHandler_CRawGameControllerVtbl_InvokeAdde
         return S_OK;
     }
 
-    hr = __x_ABI_CWindows_CGaming_CInput_CIRawGameController_QueryInterface(e, &IID_IRawGameController, (void **)&controller);
+    hr = __x_ABI_CWindows_CGaming_CInput_CIRawGameController_QueryInterface(*e, &IID_IRawGameController, (void **)&controller);
```

Comment:
These changes should be dropped. They will cause a crash when WGI invokes our callbacks on device hotplug.

Reply:
see https://github.com/libsdl-org/SDL/pull/5248#discussion_r790239592